docs/t-sql/statements/copy-into-transact-sql.md

```diff
@@ -430,9 +430,9 @@ WITH (
 ## FAQ
 
 ### What is the performance of the COPY command compared to PolyBase?
-The COPY command will have better performance depending on your workload. For best loading performance, consider splitting your input into multiple files when loading CSV. This guidance applies to gzip compressed files as well.
+The COPY command will have better performance depending on your workload. For best loading performance, consider splitting your input into multiple files when loading compressed CSV.
```

Comment:
This feels like a net loss of valuable information for customers. Does this guidance not apply to gzip compressed files? Should we specify the types of compression specified?

Reply:
@WilliamDAssafMSFT let me know if you are satisfied with this PR based on the reply from Sumin in our e-mail thread.

Reply:
I see that. Should we add "Manually splitting files is needed only for compressed CSV files."

This guidance is still a little confusing to me as someone who used to design a lot of ETL systems. Are we saying that a large uncompressed file wouldn't need to be split, but a smaller compressed file should be split up by the data source?